app/models/tutoring_session.rb

```diff
@@ -2,9 +2,11 @@ class TutoringSession < ApplicationRecord
   belongs_to :student
   belongs_to :tutor
 
-  validates :location,     presence: true
-  validates :start_time,   presence: true
-  validates :end_time,     presence: true
+  validates :location, presence: true
+  validates :hours, presence: true,
+                    length: { maximum: 4 }
```

Comment:
The max is length of the input integer, not number of hours. I put a max length of 4 because of the user put a long, I.e2748464838364748494739, the webpage will crash because the database cannot hold a long as an int.